runtime/v2/runc/service.go

```diff
@@ -78,7 +79,13 @@ func New(ctx context.Context, id string, publisher events.Publisher) (shim.Shim,
 		ep:        ep,
 	}
 	go s.processExits()
-	runcC.Monitor = shim.Default
+	rc, ok := shim.Default.(interface {
```

Comment:
could we get a public type for the interface? :)

Reply:
no ;)